std/meta.d

```diff
@@ -1543,6 +1514,8 @@ template Repeat(size_t n, TList...)
 
     alias ImInt10 = Repeat!(10, int);
     static assert(is(ImInt10 == AliasSeq!(int, int, int, int, int, int, int, int, int, int)));
+
+    alias Big = Repeat!(1,000,000, int);
```

Comment:
Am I dumb, or did you mean,

```d
alias Big = Repeat!(1_000_000, int);
```
?

Reply:
Ooops. I am dumb! Will fix tomorrow and measure again.